waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/server/FederatedHMSHandlerTest.java

```diff
@@ -1768,4 +1771,17 @@ public void get_primary_keys() throws TException {
     assertThat(result, is(expected));
   }
 
+  @Test
+  public void get_partition_values() throws TException {
+    Table table = new Table();
```

Comment:
You don't really need the intermediate `Table` object, you can just pass in the db and table names directly as arguments on line  1779.